README.md

```diff
@@ -20,9 +20,7 @@ bundle install
 touch .env
 
 # Clone production data
-rake db:create db:migrate
-rake cascade:sync
-rake faculty:sync
+cap production db:clone
```

Comment:
I feel like capistrano can also be used for server automation tasks. We need to run the `pg_dump` task on the server and with a cap task we can specify which server we want to use.